### ISSUE TITLE
Dead image links in ShowReel

### ISSUE THREAD

**Issue by alexchandel** [MauriceConrad/Photon#49]
The following image links in the demo ShowReel to https://maurice-conrad.eu/ are dead:
* https://maurice-conrad.eu/acting/images/big-narrenkaefig-1.jpg
* https://maurice-conrad.eu/acting/images/big-narrenkaefig-6.jpg
* https://maurice-conrad.eu/acting/images/big-narrenkaefig-6.jpg
* https://maurice-conrad.eu/acting/images/big-narrenkaefig-5.jpg
* https://maurice-conrad.eu/acting/images/big-schach-4.jpg

This is amusingly visible even on the demo site: https://mauriceconrad.github.io/Photon/

Please update the links, or rehost demo images directly on the demo site, or just embed some tiny SVGs in the demo.

**Comment by antoniormrzz** [comment]
Is this project dead? seems like the only interesting one

**Comment by FlashTang** [comment]
Indeed, this is the most closer project to MacOS looking  I can find

**Comment by sasmlange** [comment]
The images in the docs are also dead.
